pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -97,6 +101,8 @@ type RegistryPoll struct {
 	// The catalog operator polls to see if a new version of the catalog source is available.
 	// If available, the latest image is pulled and gRPC traffic is directed to the latest catalog source.
 	Interval *metav1.Duration `json:"interval,omitempty"`
+	// +optional
+	ParsingError string `json:"parsingError,omitempty"`
```

Comment:
nit: this type of error should be populated on the status of the catalog, as its related to the reconciliation status of the catalog object on-cluster. The spec is for users to declare what the catalog should consist of and should generally not be modified by controllers.